src/popup/router/pages/Names/AuctionList.vue

```diff
@@ -75,15 +90,61 @@ export default {
           default:
             return 1;
         }
-      }).sort(() => (this.displayMode.rotated ? -1 : 1));
+      });
     },
   },
   async mounted() {
-    this.loading = true;
     await this.$watchUntilTruly(() => this.$store.state.middleware);
-    this.activeAuctions = await this.$store.dispatch('names/fetchAuctions');
+    await this.fetchAuctionsChunk();
+    this.setScrollWatcher();
     this.loading = false;
   },
+  methods: {
+    ...mapActions('names', ['fetchAuctions']),
+    setScrollWatcher() {
+      const app = document.querySelector('#app');
+      const throttledLoadMore = () => throttle(this.checkLoadMore, 700);
```

Comment:
`throttle` shouldn't be needed, because `checkLoadMore` should do an inexpensive check in sync and to trigger async action if needed, if the action is already triggered then is should do nothing
similar to https://github.com/aeternity/superhero-ui/blob/3ad2490571b7aa2c65a4e2fc56eab3464b5f40eb/src/components/FeedPagination.vue#L71-L87